app/models/amusement_park.rb

```diff
@@ -1,3 +1,8 @@
 class AmusementPark < ApplicationRecord
   has_many :rides
+  has_many :mechanics, through: :rides
+
+  def unique_mechanics
+    mechanics.distinct
```

Comment:
Nice work adding that additional association on line 3! Makes it wayyy easier to grab the mechanics for a park